src/SFA.DAS.EmployerIncentives.Web/Services/Apprentices/IPaginationService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Threading.Tasks;
+using SFA.DAS.EmployerIncentives.Web.Services.Apprentices.Types;
+
+namespace SFA.DAS.EmployerIncentives.Web.Services.Apprentices
+{
+    public interface IPaginationService
+    {
+        Task<PagingInformation> GetPagingInformation(ApprenticesQuery apprenticesQuery);
```

Comment:
Rather than ApprenticesQuery (which is very specific) can this be an interface

e.g.
```

IPaginationQuery
{
    int PageSize { get; }
    int Offset { get; }
    int StartIndex { get; }
}

```


then ApprenticesQuery can implement the interface.

i.e.

`Task<PagingInformation> GetPagingInformation(IPaginationQuery paginationQuery);`